Tensile/Source/lib/include/Tensile/Utils.hpp

```diff
@@ -34,6 +34,8 @@
 #include <sstream>
 #include <type_traits>
 
+#define PAGE_SIZE (2097152)
```

Comment:
We have been very careful not to add any symbols outside the Tensile namespace.  This can't be a macro.